scipy/sparse/_compressed.py

```diff
@@ -1123,26 +1122,27 @@ def sum_duplicates(self):
         self.prune()  # nnz may have changed
         self.has_canonical_format = True
 
-    def __get_sorted(self):
-        """Determine whether the matrix has sorted indices
+
```

Comment:
Pedantic nitpick: there should only be one empty line between methods/properties.

Reply:
Oh, I'm so used to black doing that for me, I don't really look for it. I'll change that here.

```suggestion
```